.bazelrc

```diff
@@ -15,9 +15,8 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-build --incompatible_strict_action_env --javacopt='--release 8'
+build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11'
```

Comment:
See https://github.com/graknlabs/grakn/pull/6281 - after installing the newest Bazel (by pulling `grakn` and building it locally) I was no longer able to build `protocol`. I fixed it by upgrading Bazel and Java in protocol.